packages/custom-functions-metadata/src/parseTree.ts

```diff
@@ -619,6 +640,14 @@ function getResults(
     type: resultType,
   };
 
+  // We only support cell value types as input and not return type for now.
```

Comment:
Not sure why you need to give an error if a cell value type is put for a return value. It will still just generate "any" for it since the return type info in the metadata. 

Later it would be good to tighten this, e.g. the linked entity provider function should return `Promise<CellValue>` where CellValue is any cell value type (including error types).

Reply:
Please see my other comment, thanks.